Emby.Server.Implementations/Dto/DtoService.cs

```diff
@@ -1058,6 +1062,15 @@ private void AttachBasicFields(BaseItemDto dto, BaseItem item, BaseItem owner, D
                     dto.Chapters = _itemRepo.GetChapters(item);
                 }
 
+                if (options.ContainsField(ItemFields.Trickplay))
+                {
+                    var manifest = _trickplayManager.GetTrickplayManifest(item).GetAwaiter().GetResult();
+
+                    // To stay consistent with other fields, this must go from a Guid to a non-dashed string.
+                    // This does not seem to occur automatically to dictionaries like it does with other Guid fields.
+                    dto.Trickplay = manifest.ToDictionary(x => x.Key.ToString("N", CultureInfo.InvariantCulture), y => y.Value);
```

Comment:
Isn't this the only place where `GetTrickplayManifest` is called? Shouldn't it just return this dict?